src/sentry/integrations/slack/message_builder/notifications.py

```diff
@@ -0,0 +1,35 @@
+from sentry.integrations.slack.utils import LEVEL_TO_COLOR
+from sentry.utils.assets import get_asset_url
+from sentry.utils.http import absolute_uri
+
+
+def build_notification_attachment(notification, context):
+    logo_url = absolute_uri(get_asset_url("sentry", "images/sentry-email-avatar.png"))
+    settings_url = absolute_uri("/settings/account/notifications/")
+    activity_link = context["activity_link"]  # this has a referrer built in, change it?
```

Comment:
I can rebuild the link instead, if I use this `activity_link` it has the activity type already set as the referrer e.g. `referrer=NoteActivityEmail`. We probably want our own referrer to know the clicks are coming from Slack.